client/ayon_maya/plugins/publish/extract_thumbnail.py

```diff
@@ -74,7 +74,7 @@ def process(self, instance):
 
         representation = {
             "name": "thumbnail",
-            "ext": "jpg",
+            "ext": "png",
             "files": thumbnail,
```

Comment:
Maybe the `thumbnail` variable holds `.jpeg`?

Reply:
That's a very good point!

Reply:
Doesn't seem to be the case?

The file seems to be:
```
reviewMain.0001.png
```

As such, I think we can merge this?

Reply:
This PR fixes this warning on the latest ayon server:
```
Thumbnail mime type mismatch: Payload contains image/png but was requested to store image/jpeg
```

Reply:
So we can use `"ext": os.path.splitext(thumbnail).lstrip(".").lower()` instead of hardcoding `"png"`.

Reply:
in playblast profiles you can actually change the format:
![image](https://github.com/user-attachments/assets/d977bef2-3bec-4829-b5d0-397ed8b5cbc4)
so getting it from the filename is probably safest way?